### PR TITLE
Prevent to connect Kusama manager if there are no KSM addresses

### LINKED ISSUE #2080: Prevent SubstrateManager from connecting to public nodes unless the user has related addresses imported
As per [PR comment](https://github.com/rotki/rotki/pull/2030#discussion_r556179053)

Do it

### PR DESCRIPTION
**Pull request by vnavascues**
Closes #2080 
Also amends the changelog wording for feature #1920

### CODE REVIEW

File: rotkehlchen/chain/substrate/manager.py [hunk 1]
```diff
@@ -505,6 +497,17 @@ def _set_chain_properties(self, node_interface: SubstrateInterface) -> None:
         self.chain_properties = self._get_chain_properties(node_interface)
         return None
 
+    def attempt_connections(self) -> None:
```
Comment (vnavascues) [inline rotkehlchen/chain/substrate/manager.py:500]: Now this this method is "public", as it is called for kicking off the connections, for instance when adding the first KSM address.

File: rotkehlchen/chain/substrate/manager.py [hunk 2]
```diff
@@ -607,3 +610,22 @@ def set_rpc_endpoint(self, endpoint: str) -> Tuple[bool, str]:
             self.own_rpc_endpoint = endpoint
 
         return result, message
+
+
+def wait_until_a_node_is_available(
```
Comment (vnavascues) [inline rotkehlchen/chain/substrate/manager.py:615]: Moved into the manager.py file as it is a "utils" (we use it at least twice).

**Review by LefterisJP — APPROVED** [review]
Have not tried that it would work as expected but I can't see something wrong. So approving and merging. Thank you.

**Comment by vnavascues** [comment]
There is a test specific for it (add a new KSM account when db has none) and it works. I could be possible that sometimes 5s wouldn't be enough and timeouts... we will see. I've never had a problem within 5s.

**Pull request merged**